src/posts/how-to-use-JAWS-on-Windows

```diff
@@ -0,0 +1,47 @@
+- H: Pressing the H key moved you between headings.
+- INSERT + DOWN ARROW: This keyboard shortcut will allow you to read the current line in the website.
+- INSERT + UP ARROW: This keyboard shortcut will allow you to read from the current position to the top of the document or application.
+- ALT + TAB: This keyboard shortcut will allow you to switch between open applications or windows.
+- WINDOWS KEY + M: This will minimize all windows, allowing you to access the desktop quickly.
+- Control + F: This key allows you to open the search feature within your browser.
+Listed here are some basic keyboard shortcuts you can use to start using JAWS right away. For a full list of shortcut commands, you can visit [JAWS website](https://doccenter.freedomscientific.com/doccenter/doccenter/rs25c51746a0cc/2011-10-04_jawsandmagicreadingcommands/JAWSKeystrokes.htm). JAWS provide many keyboard shortcuts for a variety of reasons. In the next section, we will explore some advanced features JAWS provide to their users. 
+
+## Advanced features
+JAWS is an incredible screen reader that has many advanced features you can explore. On top of providing a great screen reader experience, below are some advanced features you can try to customize JAWS.
+- INSERT + J: This command opens JAWS settings. After opening JAWS settings, you can navigate through the various options to customize JAWS. Some options include:
+-- Adjusting verbosity level to determine the amount of detail JAWS provides when reading text.
+-- Customize punctuation settings to control how JAWS pauses and pronounces words. 
+-- Voice Adjustment" section. Here, you can modify voice speed, pitch, volume, and other voice-related settings to suit your preferences.
+
+- INSERT + SPACE followed by the letter O: This keyboard command will open OCR, which is Optical Character Recognition. Move your cursor over the desired text and JAWS will read it aloud to you. OCR can be extremely helpful for accessing content in image-based PDFs. 
+- INSERT + SPACE followed by V: Some applications may not be fully accessible to screen readers like JAWS. In those cases, we can use the feature called “Virtualizing”. This will allow JAWS to interact with the application’s interface more effectively, allowing you to navigate and access content within the application.
+
+## Conclusion
+JAWS is an incredible screen reader that empowers individuals with visual impairments to access and interact with computers running the Windows operating system. By following this comprehensive guide, you can download and use JAWS to navigate applications, read documents and browse the web. You can first start using JAWS basic commands to navigate through the web and applications. In time, you can explore advanced features and customize JAWS with your own shortcuts. JAWS is an incredible tool that opens the web and a world of tech to people with visual impairments.
```

Comment:
You sort of repeat your self with "JAWS is an incredible screen reader" and "explore advanced features". From a summarizing what you wrote about perspective thought it's fine, but I sort of skip this because I've already read it and it reads a bit like marketing copy.